examples/remote_procedure/mnist/run_task.py

```diff
@@ -58,12 +79,24 @@ def handle_with_model(
     function_registry = {
         "classify_digit": handle_with_model,
     }
-
-    shared_state = SharedRemoteProcedureTaskState(
-        static_task_data=tasks,
-        function_registry=function_registry,
-    )
-
+    if is_using_screening_units:
+        shared_state = SharedRemoteProcedureTaskState(
+            static_task_data=tasks,
+            function_registry=function_registry,
+            on_unit_submitted=ScreenTaskRequired.create_validation_function(
+                cfg.mephisto,
+                validate_screening_unit,
+            ),
+            screening_data_factory=my_screening_unit_generator(),
+        )
+        shared_state.qualifications += ScreenTaskRequired.get_mixin_qualifications(
+            cfg.mephisto, shared_state
+        )
```

Comment:
this is unrelated to this PR, but it seems that `ScreenTaskRequired.get_mixin_qualifications` is not even using `shared_state` despite requiring it in the signature.

Reply:
`get_mixin_requirements` has a shared signature for all `BlueprintMixin`s, as ideally this will be replaced with `<>Blueprint.get_mixin_qualifications` automatically pulling required qualifications into the state _under the hood_.